lightning/config/config.go

```diff
@@ -229,11 +230,17 @@ func (cfg *Config) Load() error {
 		}
 	}
 
-	// If the level 1 compact configuration not found, default to true
+	// If the level 1 compact configuration not found, default to false
 	if cfg.PostRestore.Level1Compact == nil {
 		cfg.PostRestore.Level1Compact = new(bool)
-		*cfg.PostRestore.Level1Compact = true
+		*cfg.PostRestore.Level1Compact = false
 	}
 
+	if cfg.App.TableConcurrency < 2 {
+		cfg.App.TableConcurrency = 2
+	}
+	if cfg.App.IndexConcurrency < 2 {
+		cfg.App.IndexConcurrency = 2
+	}
```

Comment:
I don't think you need to mandate the minimal concurrency at all

Reply:
adressed